microprofile/telemetry/pom.xml

```diff
@@ -142,6 +142,20 @@
                         </path>
                     </annotationProcessorPaths>
                 </configuration>
+                <executions>
+                    <execution>
+                        <id>default-testCompile</id>
+                        <configuration>
+                            <compilerArgs>
+                                <!--
+                                Used by a test to send requests with repeating header names.
+                                -->
+                                <arg>--add-modules</arg>
```

Comment:
Just a comment, not a request for a change; it's too minor.

This is odd. I understand why it's here.

Maybe there's a way to run `testCompile` "without modules", in much the same way you can run Surefire "without modules" (via `useModulePath=false`). Otherwise this is a little leaky: there are other `testCompile` executions that could bypass this (I haven't tried it but I'd be willing to bet that something like a direct invocation of `compiler:testCompile` will trigger the `default-cli` execution and bypass this).

Reply:
Aha, yes there is: https://maven.apache.org/plugins/maven-compiler-plugin/testCompile-mojo.html#useModulePath